src/status_im/contexts/wallet/swap/events.cljs

```diff
@@ -9,11 +10,18 @@
  (fn [{:keys [db]} token]
    {:db (assoc-in db [:wallet :ui :swap :asset-to-pay] token)
     :fx [[:dispatch
-          [:toasts/upsert
-           {:id   :swap-error
-            :type :negative
-            :text "Not implemented yet"}]]]}))
+          [:navigate-to :screen/wallet.swap-propasal]]
+         [:dispatch [:wallet.swap/init]]]}))
 
 (rf/reg-event-fx :wallet.swap/clean-asset-to-pay
  (fn [{:keys [db]}]
    {:db (update-in db [:wallet :ui :swap] dissoc :asset-to-pay)}))
+
+(rf/reg-event-fx :wallet.swap/init
+ (fn [{:keys [db]}]
+   {:db
+    (assoc-in db [:wallet :ui :swap :max-slippage] constants/default-slippage)}))
+
+(rf/reg-event-fx :wallet.swap/set-max-slippage
+ (fn [{:keys [db]} [max-slippage]]
+   {:db (assoc-in db [:wallet :ui :swap :max-slippage] (js/parseFloat max-slippage))}))
```

Comment:
I would say it's not a good idea to use `parseFloat` directly because if this function is given a value it can't parse, such as nil or an empty string, it will return `NaN`. And nils are everywhere in the runtime. I lost count of the amount of apps written in JS with `NaN` being displayed to users and leading to wrong calculations on the client.

A better idea is to do something similar to `utils.number/parse-int` and have our own `parse-float` that returns a sensible default float when the value can't be parsed instead of `NaN`, such as zero.

The only difference from `parse-int` is that we would need to use `js/Number.isNaN` instead of `integer?` to check the parsed result.

But of course, although my suggestion is simple to implement, you may consider outside the scope of this PR :)